src/scripts/admin/blocks/collapse/container/deprecated.js

```diff
@@ -0,0 +1,41 @@
+import { __ } from '@wordpress/i18n';

```

Comment:
you know you don't need deprecated, unless you have stuff in production... you probably could get rid of this file.

Reply:
Good call.